FinSearchUnified/Resources/services.xml

```diff
@@ -12,16 +12,23 @@
     <services>
         <service id="fin_search_unified.subscriber.frontend" class="FinSearchUnified\Subscriber\Frontend">
             <argument>%fin_search_unified.plugin_dir%</argument>
-            <argument type="service" id="template" />
-            <tag name="shopware.event_subscriber" />
+            <argument type="service" id="template"/>
+            <tag name="shopware.event_subscriber"/>
+        </service>
+
+        <service
+            id="fin_search_unified.sdym_criteria_request_handler"
+            class="FinSearchUnified\Bundle\SearchBundle\CriteriaRequestHandler\SdymCriteriaRequestHandler"
+        >
+            <tag name="criteria_request_handler"/>
```

Comment:
Separate the double quote and slash by a single white space.

Reply:
The change described in the previous comment didn't happen.

Reply:
My bad, I thought the revert of this file would fix this.